docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -469,9 +471,9 @@ Fields:
 (note: all fields can be expanded by project variables with the exception of `auto_update`)
 
 -   `name`: alias to refer to the module
--   `branch`: the branch of the module to use in the project
+-   `branch`: the branch of the module to use in the project. These can be [expansions](Project-Configuration-Files#expansions)
 -   `repo`: the git repository of the module
--   `prefix`: the path prefix to use for the module
+-   `prefix`: the path prefix to use for the module. These can be [expansions](Project-Configuration-Files#expansions)
```

Comment:
I added these expansion links so user's better know what is allowed, I originally was trying to see where we document that the prefix _can_ be an expansion